doc/topics/cloud/vmware.rst

```diff
@@ -146,10 +146,7 @@ Set up an initial profile at ``/etc/salt/cloud.profiles`` or
             name: 10.40.50-600-Prod
             adapter_type: vmxnet3
             switch_type: distributed
-            ip: 10.40.50.123
-            gateway: [10.40.50.110]
-            subnet_mask: 255.255.255.128
-            domain: example.com
```

Comment:
I figured that was the reason but wanted to check with you. I think we're fine to merge this now. Thanks!